app/views/static/generic_object/main_custom_button_form.html.haml

```diff
@@ -36,7 +36,7 @@
           = "<#{_('Choose')}>"
       %span.help-block{"ng-show" => "angularForm.dialog_id.$error.required"}
         = _("Required")
-  .form-group{"ng-class" => "{'has-error': vm.angularForm.display.$invalid}"}
+  .form-group{"ng-class" => "{'has-error': angularForm.display.$invalid}"}
```

Comment:
although `angularForm.display.$invalid` is correct in this case, the error would never be exposed in this field which is a checkbox with Yes/No values - so technically, there is no "invalid" value here.
Anyway, good to have this change.